rootio/cmd/root-srv/main.go

```diff
@@ -13,22 +14,28 @@
 //  $> root-srv -addr=:8080 &
 //  2017/04/06 15:13:59 server listening on :8080
 //
+
```

Comment:
please remove that blank line (otherwise the documentation for the `root-srv` command in godoc only reads `$> open localhost:8080` :))